common/scripted_triggers/00_coa_triggers.txt

```diff
@@ -4,6 +4,14 @@ coa_african_trigger = {
 	OR = {
 		coa_african_culture_trigger = yes
 		coa_african_religion_trigger = yes
+		
+		#Warcraft
+		scope:culture = {
+			OR = {
+				has_graphical_culture = harpy_coa_gfx
+				has_graphical_culture = troll_coa_gfx
+			}
+		}
```

Comment:
Why not to use it in `coa_african_culture_trigger` since it's created for cultures?